lib/Tools/GenericSwizzling.cpp

```diff
@@ -1,4 +1,5 @@
 #include "triton/Tools/GenericSwizzling.h"
+#include "triton/Conversion/TritonGPUToLLVM/TargetInfoBase.h"
```

Comment:
Could this cause an circular dependency, as `TritonGPUIR` relies on `TritonTools`?

Reply:
Good point

Reply:
Thanks for fixing this! Can this file include be removed then?

Reply:
ugh, yeah, forgot to remove this include. let me put up a PR

Reply:
https://github.com/triton-lang/triton/pull/7700